packages/plugin-compiler/src/main/kotlin/io/realm/compiler/RealmSchemaLoweringExtension.kt

```diff
@@ -246,31 +256,25 @@ private fun populateCompanion(
     pluginContext: IrPluginContext
 ): IrCallImpl {
     if (specifiedModels.isEmpty()) {
-        logError("Schema argument must be a list of class literal (T::class)")
+        logError(
+            "No schema was provided. It must be defined as a set of class literals (MyType::class) either through " +
+                "RealmConfiguration(schema = setOf(...)), RealmConfiguration.Builder(schema = setOf(...).build(), " +
+                "or RealmConfiguration.Builder().schema(...).build()."
```

Comment:
varargs is not supported

Reply:
I added an override in the builder where it is supported (as the only place).